packages/create-vite/src/index.ts

```diff
@@ -35,10 +36,11 @@ const helpMessage = `\
 Usage: create-vite [OPTION]... [DIRECTORY]
 
 Create a new Vite project in JavaScript or TypeScript.
-With no arguments, start the CLI in interactive mode.
+If it is run in TTY, the CLI will start in interactive mode.
```

Comment:
I think it should be "running" here:

```suggestion
If it is running in TTY, the CLI will start in interactive mode.
```

Or maybe it can be shorter like

```suggestion
When running in TTY, the CLI will start in interactive mode.
```